scripts/install-env.sh

```diff
@@ -60,6 +60,11 @@ else
     curl https://sh.rustup.rs -sSf | sh
 fi
 
+if [ -f ~/.bash_profile ]
+then
+    export PATH=$PATH:$HOME/.cargo/bin
```

Comment:
This will work for this session, but if we want `~/.cargo/bin` to stay on the PATH next time the terminal is launched, we need to  this add the line _to_ the `.bash_profile`. You could use 
```bash
echo "export PATH=$PATH:$HOME/.cargo/bin" >> ~/.bash_profile
```
or something.